src/pg_timestampz.erl

```diff
@@ -19,4 +20,4 @@ decode(Bin, _TypeInfo) ->
     pg_timestamp:decode_timestamp(Bin, []).
 
 type_spec() ->
-    pg_timestamp:type_spec().
+    "{{Year::integer(), Month::1..12, Day::1..31}, {Hours::0..23, Minutes::0..59, Seconds::0..59 | float()}, {HourOffset::-15..15, MinuteOffset::0..59}}".
```

Comment:
This is kind of a best guess based on `pg_timestamp:type_spec`

Reply:
Did a quick search and are you sure its not `-12..14` for hours?